.github/workflows/iam-role-policy-changes-check.yaml

```diff
@@ -0,0 +1,29 @@
+name: Identify PRs that contain IAM Role and Policy changes
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+id: review_pr
+env:
+   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+jobs:
+  iam-role-policy-changes-check.yml:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
You will need something like
```
     - name: Checkout PR code
        uses: actions/checkout@master
     - name: Run iam/role policy changes check
        uses: ministryofjustice/github-actions/iam-role-policy-changes-check@iam-role-policy-changes-check
```

Reply:
Hi @poornima-krishnasamy Poornima
Hopefully have corrected the above?